cfgov/v1/models/molecules.py

```diff
@@ -31,6 +33,32 @@ class ImageText5050(blocks.StructBlock):
     link_url = blocks.URLBlock(required=False)
     link_text = blocks.CharBlock(max_length=100, required=False)
 
+    def clean(self, data):
+        error_dict = {}
+        try:
+            block_data = super(ImageText5050, self).clean(data)
+        except ValidationError as e:
+            error_dict.update(e.params)
+            block_data = data
```

Comment:
nah it doesn't need to be converted. you were right about accessing the variables from an orderedDict like so `block_data['image']` without having to convert it.